packages/model/src/research-output.ts

```diff
@@ -203,6 +205,7 @@ export type ResearchOutputDataObject = ResearchOutputCoreObject & {
   organisms: string[];
   subtype?: string;
   teams: Pick<TeamResponse, 'id' | 'displayName'>[];
+  publishingEntity?: ResearchOutputPublishingEntities;
```

Comment:
What happens if this isn't set? I think ideally there would be a migration that sets all existing RO to Team.